lib/linguist/languages.yml

```diff
@@ -2905,6 +2905,18 @@ NewLisp:
   codemirror_mode: commonlisp
   codemirror_mime_type: text/x-common-lisp
   language_id: 247
+Nextflow:
+  type: programming
+  ace_mode: groovy
+  tm_scope: source.nextflow
+  color: "#3ac486"
+  extensions:
+  - ".nf"
+  filenames:
+  - "nextflow.config"  
```

Comment:
Can you please add a good illustrative sample of a `nextflow.config` file to `samples/Nextflow/filenames/` too.

Another question: would one normally want this file to count towards the language statistics? If not, you may want to mark it as vendored too.

Reply:
Also, since `nextflow.config` has not the same syntax as nextflow script I guess it should not be included in the `filenames`, should it?

Reply:
It really depends if it should be counted in the Nextflow language stats or not. If not, then no, we don't need the entry here and its language will be determined as per normal, unless marked as vendored or generated.

Reply:
It think it's fine to include in the language stats.

Reply:
:cool: Now would people expect it to be classified as "Nextflow" or the actual language it's written in? If the latter, what is that language? It appears to be a very similar to JSON from the few I've seen.

Reply:
It's a customisation of the Groovy ConfigObject syntax (hence I would propose to classify it as Nextflow)

Reply:
:+1: In that case, we just need an illustrative sample of a `nextflow.config` file added to `samples/Nextflow/filenames/` and we should be good to go.

Reply:
And by "illustrative sample", I mean a good real-world-usage file that demonstrates the usage of the file well.

Reply:
Sure thing. The one just uploaded should be illustrative enough.